items.h

```diff
@@ -207,6 +209,14 @@ enum {
 };
 
 
+// Describes bonus/mauls against another weapon
+class WeaponBonusMalus {
```

Comment:
as all other my comments -- its just a comment.
But in all x64 size of the word is 8, and due to pointer in structure, it will be anyway aligned to 8.
So its nice to use `int32_t attackModifer; int32_t defenseModifer` so structure size will be 16, not 24.

Reply:
I will not do that, mainly because I cannot predict what will outcome (not C++ expert) and 8 bytes per one item won't be noticeable as the weapon list is fixed and it is a singleton.